src/frontend/src/icons/linear/linear.jsx

```diff
@@ -0,0 +1,30 @@
+const Icon = (props) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    width="23px"
+    height="23px"
+    viewBox="0 0 23 23"
+    version="1.1"
```

Comment:
```suggestion
    version="1.1"
    fill="currentColor"
    {...props}
```

This will support light/dark mode